### ISSUE TITLE
A question of downstream task

### ISSUE THREAD

**Issue by Shuanglin-1126** [facebookresearch/sapiens#174]
My research focuses on sign language recognition. I use sapiens' image encoder to extract image features, then perform time series modeling on the features and complete the classification work. I've found that image encoders don't work as well as CLIP's. But due to the dataset used in the pre-training phase is Humans-300M, I think the sapiens should work better. Do you have any suggestions for the feature processing task.

**Comment by Shuanglin-1126** [comment]
I have tried two schemes, the first scheme is to perform one convolution and five pooling of the output of 0.3B sapiens (1024,64,64) to obtain (512,1,1) features, and to model and classify them through 1D-CNN and BiLSTM.The second scheme is to design a decoding layer, use a class token to calculate the cross-attention with the features of the final output of sapiens, and finally obtain the features of the class token after passing through the three layers of decoding layers, and carry out timing modeling and classification. Both schemes were found to have no convergence and were lower than the accuracy of using CLIP. (Although the results of the clip are also very unsatisfactory)

**Comment by rawalkhirodkar** [comment]
@Shuanglin-1126 thank you for your interest in the project.
I would recommend using all the features with convolution and pooling instead of the class token approach.

- When you compare CLIP vs Sapiens, are you using same number of parameters? Sapiens features are denser and low-level, they may require more compute for classification.
- I would also recommend changing the backbone to 0.6B or 1B if you have the compute. We observe increased generalization abilities as the model size increases.

**Comment by Shuanglin-1126** [comment]
@rawalkhirodkar Thanks for your reply, I will consider your suggestion to increase the network complexity, but the inference time of 0.6b is too long, please ask if your team would be interested in adjusting the input size to 224 and pre-training again？

**Comment by rawalkhirodkar** [comment]
@Shuanglin-1126 unfortunately, there are no plans of pretraining at lower resolutions. 
However, finetuning at lower resolutions should also result in descent results, we have not thoroughly benchmarked this.
